spec.md

```diff
@@ -1827,6 +1957,12 @@ operand types are valid and how to convert the operand `x` to a string:
 s       any             as if by str(x)
 r       any             as if by repr(x)
 d       number          signed integer decimal
+o       number          signed octal
+e       number          float exponential format, lowercase (1.230000e+12)
+E       number          float exponential format, uppercase (1.230000E+12)
+f       number          float decimal format                (1230000000000.000000)
+g       number          compact format, lowercase           (0.0, 1.1, 1200, 1e+45, 1.2e+12) 
+G       number          compact format, lowercase           (0.0, 1.1, 1200, 1e+45, 1.2E+12)
 ```
 
 It is an error if the argument does not have the type required by the
```

Comment:
Except ints can be promoted to floats and floats can be truncated to ints.

Reply:
Done.